charts/monitoring/grafana/config/grafana.ini

```diff
@@ -17,3 +17,7 @@ auto_sign_up = true
 [users]
 viewers_can_edit = {{ .Values.grafana.provisioning.configuration.viewers_can_edit }}
 auto_assign_org_role = {{ .Values.grafana.provisioning.configuration.auto_assign_org_role }}
+{{ if .Values.grafana.provisioning.configuration.root_url }}
+[server]
+root_url = {{ .Values.grafana.provisioning.configuration.root_url | quote }}
+{{ end }}
```

Comment:
Can you keep the trailing newline at the end of the file?

You could also use `with` instead of `if` to reduce the code duplication, but that's just nitty picky.

Reply:
Yeah frankly realised the missing newline after I pushed it. fixed now.